oktoberfest/predict/predict.py

```diff
@@ -134,11 +137,10 @@ def _alignment(alignment_library: Spectra):
 
     :param alignment_library: the library to perform the alignment on
     """
-    pred_intensity = alignment_library.get_matrix(FragmentType.PRED)[0]
-    raw_intensity = alignment_library.get_matrix(FragmentType.RAW)[0]
-    # return pred_intensity.toarray(), raw_intensity.toarray()
+    pred_intensity = alignment_library.get_matrix(FragmentType.PRED)[0].toarray()
+    raw_intensity = alignment_library.get_matrix(FragmentType.RAW)[0].toarray()
```

Comment:
do we really need to change to array first? I thought it is taken care of automatically. Maybe check in spectrum_fundamentals

Reply:
I looked it up. Spectrum fundamentals also accepts sparse matrices, so I removed the conversion.